src/styles/header.scss

```
@@ -0,0 +1,67 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  max-height: 60px;
+  box-sizing: border-box;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  display: block;
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+  font-weight: 500;
+  font-size: 12px;
+  text-transform: uppercase;
+  line-height: 14px;
+  display: flex;
+}
```

```suggestion
.nav{
 &__list {
  margin: 0;
  padding: 0;
  font-weight: 500;
  font-size: 12px;
  text-transform: uppercase;
  line-height: 14px;
  display: flex;
 }
}
```
pls make use of sass nesting for bem everywhere in your project